src/views/Board.vue

```diff
@@ -93,6 +93,10 @@ export default {
         }
       ]
     };
+  },
+  //makes it so that when taken to page from router, it goes to the top
```

Comment:
hmm...why is this needed? This seems like an indirect solution to what might be a deeper-rooted problem

Reply:
read my thread with jack

Reply:
this is the solution that applicable, no deeper rooted problem

Reply:
lgtm